src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from "../CommentList";
+import './PostInfo.scss';
+import { UserInfo } from "../UserInfo";
+
+export const PostInfo = ({ post }) => (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {post.title}
+        </h3>
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={post.user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {post.body}
+      </p>
+
+      {Object.keys(post.comments).length !== 0 ? (
```

Comment:
You are checking if an object has keys, which implies that you are expecting an object. However, it would be more appropriate to expect an array of comments for a post. You may want to revise the data structure and check if the array length is 0 for absence of comments.